packages/eslint-plugin-gestalt/rollup.config.js

```diff
@@ -11,14 +11,14 @@ const rollupConfig = {
       format: 'umd',
       name: 'eslint-plugin-gestalt',
       exports: 'named',
-      sourcemap: 'inline',
+      sourcemap: false,
```

Comment:
set this to false bc i don't think we need to generate sourcemaps for the eslint-plugin, but happy to also just set this to true